doc/Customization-guide/How-to-contribute/index.md

```diff
@@ -7,27 +7,42 @@ sort_idx: 50
 
 {% include variables.md %}
 
-We welcome you to join our rapidly growing community!
+* TOC
+{:toc}
 
-As an open-source project, we thrive from contributions by people like you to create the best possible platform for developing IoT solutions. We would love to see you mastering Kaa's source code, however, writing code is not the only way to contribute. There are many other options, such as providing ideas, suggestions and comments in [Kaa forum](https://groups.google.com/forum/#!forum/kaaproject) discussions, testing features and new releases, and reviewing and improving the documentation. Every contribution counts -- below you will find more information on how you can play a role.
+Welcome to Kaa community!
 
-## Joining the forum
-You can join our [forum](https://groups.google.com/forum/#!forum/kaaproject) to provide feedback, make features and functionality suggestions, ask questions, help other community members, or get in touch with our engineering team.
+As an open-source IoT solution, [Kaa platform]({{root_url}}Glossary/#kaa-platform) thrives from contributions by people like you.
+We would love to see you mastering the Kaa source code, however, writing code is not the only way you can contribute to Kaa.
 
-## Testing releases or issuing bug reports
+Feel free to use all of the contribution options:
 
-You can report your findings by creating a bug in [Jira®](http://jira.kaaproject.org/browse/KAA/). Please provide as much detail as possible so that any identified issues can be effectively resolved.
+- Join [Kaa StackOverflow](http://stackoverflow.com/questions/tagged/kaa) to provide feedback, make features and functionality suggestions, ask questions, help other community members, and get in touch with our engineering team.
```

Comment:
StackOverflow is for questions only. Feedback, feature requests, and suggestions are banned there – they should go to Jira.